docs/specifications/tests/Zone-TP/zone09.md

```diff
@@ -1,47 +1,285 @@
+* [Terminology](#terminology)
+
+
+## Objective
+
+It is strongly recommended in [RFC 2142][RFC 2142#section-7],
+section 7, that every domain should have a mailbox named
+hostmaster@DOMAIN (where DOMAIN is *Child Zone* in this test case).
 
 > For simplicity and regularity, it is strongly recommended that the
 > well known mailbox name HOSTMASTER always be used
 > \<HOSTMASTER@domain\>.
 
-It is therefore necessary for any domain to publish an MX record.
-However, SMTP can make a delivery without the MX, using the A or
-AAAA record as specified in section 5.1 of
-[RFC 5321](https://tools.ietf.org/html/rfc5321#section-5.1).
+It is therefore expected for every domain (zone), excluding some cases described
+below above, to publish an MX record in apex of the zone (i.e. in the same
+position as the SOA record). If MX is not present, SMTP *can* deliver email using
+an address record (A or AAAA) as specified in [RFC 5321][RFC 5321#section-5.1],
+section 5.1, but that possibility is not in common use. This test case only
+checks for MX record and ignores the possibility to use address records for email
+since the absolutely dominating way to configure support for email is to use MX.
+If an address record is found in apex it is more often used for web service rather
+than email service.
 
-### Inputs
+Even if not mentioned in [RFC 2142], there are some exception to the
+rule above.
 
-The domain name to be tested.
+The purpose of zone in the .ARPA tree is to hold infrastructural identifier,
+and it is not expected that those are used as email domains, i.e. being at the
+right of the at-sign ("@") in an email address ([RFC 3172]). This also means that
+the well known mailbox is not expected for reverse zones (zone under in-addr.arpa
+or ip6.arpa). Such zone are therefore excluded from the requirement in this test
+case.
 
-### Ordered description of steps to be taken to execute the test case
+The root zone cannot be an email domain since the email domain is
+the part to the left of the trailing dot, e.g. "example.com" is the
+email domain for the apex of "example.com.". The root zone is
+excluded from the requirement.
 
-1. Query the MX record from a delegated name server for the domain.
-2. If the answer from step 1 is not authoritative, iterate step 1 until there is an authoritative answer.
-3. If there are MX records present in the answer, this part of the test
-   case succeeds.
-4. If there are no MX records present in the answer from the MX query,
-   query for the A and AAAA records for the domain.
-5. If there is any A och AAAA record in the answers from the queries in
-   step 4, this test case succeeds.
+Top-level domains ([TLDs][TLD]) can technically function as email domains
+([RCF 5321][RFC 5321#section-2.3.5], section 2.3.5) but they rarely have that
+function and are probably not meant to be included in the specification in
+[RFC 2142]. On the contrary, [Internet Architecture Board] concludes in a report
```

Comment:
Do we really need to mention RFC 2142 here? Isn't it overruled by the newer RFC 5321?
```suggestion
function. The [Internet Architecture Board] concludes in a report
```

Reply:
It does not say so.

Reply:
I'm sorry, I don't follow. What does not say what?

Reply:
Updated.